UPGRADE.md

```diff
@@ -1,5 +1,11 @@
 # Upgrade
 
+## 1.0.0-RC6
+For the compatibility with `doctrine/orm ^2.6` the function names of the FromRepository are edited to avoid inheritance issues.
```

Comment:
add an empty line before and after this line. and there is a small typo in `Form...` (From...)